src/Sarif/Core/Stack.cs

```diff
@@ -102,7 +102,7 @@ public static Stack Create(string stackTrace)
 
             stack.Frames = new List<StackFrame>();
 
-            var regex = new Regex(StackFrame.AT + @"([^)]+\))(" + StackFrame.IN + "([^:]+:[^:]+)" + StackFrame.LINE + " (.*))?", RegexOptions.Compiled);
+            var regex = new Regex(StackFrame.AT + @"([^)]+\))(" + StackFrame.IN + @"([^:]+:[^:]+|\/.+)" + StackFrame.LINE + " (.*))?", RegexOptions.Compiled);
```

Comment:
>Regex [](http://example.com/codeflow?start=28&length=5)

Perhaps this specific issue warrants its own PR?

Reply:
Agree, created one and reverted this change.
[https://github.com/microsoft/sarif-sdk/issues/2451](https://github.com/microsoft/sarif-sdk/issues/2451)